openshift/kustomize/overlays/openshift/resources/catalogs/certified-operators.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterCatalog
+metadata:
+  name: certified-operators
+spec:
+  source:
+    type: Image
+    image:
+      pollInterval: 10m
+      ref: registry.redhat.io/redhat/certified-operator-index:v4.18
```

Comment:
Not for right now, but something to consider in the future. I am curious if we could template the OCP version so that we would no longer need to manually bump this.

We would need to figure out what that would mean for catalog bootstrapping timing because we would want to make sure the templated catalog reference always exists.

Reply:
Maybe, but is this any different than having the ART bot do these bumps for us? I don't know what we would need to do to enable that, but it seems reasonable to me that we work with ART to get the same system in place here that we have in operator-marketplace for bumping the catalog image tags.